packages/blockchain-api/app.alfajores.yaml

```diff
@@ -7,6 +7,6 @@ env_variables:
   # Pull addresses from the build artifacts of the network in protocol/build
   CELO_GOLD_ADDRESS: '0x4813BFD311E132ade22c70dFf7e5DB045d26D070'
   CELO_DOLLAR_ADDRESS: '0x299E74bdCD90d4E10f7957EF074ceE32d7e9089a'
-  FAUCET_ADDRESS: '0x456f41406B32c45D59E539e4BBA3D7898c3584dA '
+  FAUCET_ADDRESS: '0xcea3ef8e187490a9d85a1849d98412e5d27d1bb3'
```

Comment:
FYI this may change again today after we re-deploy. Check with @timmoreton or @asaj